torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -315,5 +315,96 @@ struct TORCH_API GumbelSoftmaxFuncOptions {
 
 } // namespace functional
 
+// ============================================================================
+
+/// Options for MultiheadAttention functional and module.
+struct TORCH_API MultiheadAttentionOptions {
+  MultiheadAttentionOptions(int64_t embed_dim, int64_t num_heads)
```

Comment:
I think we should move this constructor to options/activation.cpp for consistency